packages/webapp/src/containers/Finances/CustomRevenueType/EditCustomRevenue.jsx

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import PureSimpleCustomType from '../../../components/Forms/SimpleCustomType';
+import { HookFormPersistProvider } from '../../hooks/useHookFormPersist/HookFormPersistProvider';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { updateCustomRevenueType } from '../saga';
+import { revenueTypeByIdSelector, revenueTypesSelector } from '../../revenueTypeSlice';
+import { CUSTOM_REVENUE_NAME } from './constants';
+import { hookFormUniquePropertyValidation } from '../../../components/Form/hookformValidationUtils';
+
+function EditCustomExpense({ history, match }) {
+  const { revenue_type_id } = match.params;
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const selectedCustomRevenueType = useSelector(revenueTypeByIdSelector(revenue_type_id));
```

Comment:
`revenue_type_id` will be a string after `match.params` and this `id` (unlike the uuid) is a number and it won't work in the selector

Reply:
Thank you 🙏